ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthGetCode.java

```diff
@@ -64,7 +64,8 @@ protected BlockParameterOrBlockHash blockParameterOrBlockHash(
   @Override
   protected String resultByBlockHash(final JsonRpcRequestContext request, final Hash blockHash) {
     final Address address = request.getRequiredParameter(0, Address.class);
-    if (GoQuorumOptions.goQuorumCompatibilityMode && privacyParameters.isPresent()) {
+    final boolean isGoQuorumCompatibilityMode = GoQuorumOptions.goQuorumCompatibilityMode;
+    if (isGoQuorumCompatibilityMode && privacyParameters.isPresent()) {
```

Comment:
Same here, can we presume if `privacyParameters` is set we need goQuorum handling?  Or is there a goQuorum sub option in `privacyParametes`?